admin/server/projects.go

```diff
@@ -317,7 +333,7 @@ func (s *Server) ListProjectMembers(ctx context.Context, req *adminv1.ListProjec
 	}
 
 	claims := auth.GetClaims(ctx)
-	if !claims.Can(ctx, proj.OrganizationID, auth.ReadOrgMembers, proj.ID, auth.ReadProjectMembers) {
+	if !claims.ProjectPermissions(ctx, proj.OrganizationID, proj.ID).ReadProjectMembers {
```

Comment:
now an org viewer will not be able to list project members, is that correct ?

Reply:
makes sense, however we add project viewer role to the all_user_group of the org upon project creation.